packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,116 @@
+:local {
+  .control {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: inline-block;
+    font-family: inherit;
+    margin-right: var(--terra-form-checkbox-margin-right, 2em);
+    margin-top: var(--terra-form-checkbox-margin-top, 0.625em);
+    vertical-align: middle;
+
+    .label-text {
+      font-size: var(--terra-form-checkbox-font-size, 1rem);
+    }
+
+    /* Hide actual input element */
+    input:not(:checked),
```

Comment:
I'm confused by the need for these selectors. Why wouldn't you just add a class to your input and use that?